packages/core/src/components/QueryBuilder/QueryBuilder.tsx

```diff
@@ -50,6 +50,8 @@ export interface HvQueryBuilderProps {
   readOnly?: boolean;
   /** Renderers for custom attribute types. */
   renderers?: HvQueryBuilderRenderers;
+  /** Whether to opt-out of the confirmation dialogs shown before removing rules and rule groups. Default to `false`. */
+  unrestricted?: boolean;
```

Comment:
`unrestricted` feels a little generic - when I read it, it's not clear what it does

> Whether to **opt-out of the confirmation** dialogs

Inferring from the docs... what do you think about `disableConfirmation`?

Reply:
Yup, it seems better! Updated ✅